src/polymer/polymer-element.ts

```diff
@@ -43,8 +44,78 @@ export interface BasePolymerProperty {
 
 export interface ScannedPolymerProperty extends ScannedProperty,
                                                 BasePolymerProperty {}
+
 export interface PolymerProperty extends Property, BasePolymerProperty {}
 
+export function mergePropertyDeclarations(
+    propA: Readonly<ScannedPolymerProperty>,
+    propB: Readonly<ScannedPolymerProperty>): ScannedPolymerProperty {
+  if (propA.name !== propB.name) {
+    throw new Error(
+        `Tried to merge properties with different names: ` +
+        `'${propA.name}' and ' ${propB.name}'`);
+  }
+  const name = propA.name;
+  let description;
+  if (propA.description !== undefined && propB.description !== undefined) {
+    description = propA.description.length < propB.description.length ?
```

Comment:
Moved to jsdoc. It duplicates some of `jsdoc.join(...)`. I remember that there was a pretty good argument for that logic.. I think. But this looks correct for this case. Left a comment to investigate.